frontend/sass/components/_modal.scss

```diff
@@ -0,0 +1,61 @@
+@use "../abstracts" as abstracts;
+
+.modal-background {
+  display: flex;
+  position: fixed;
+  background-color: rgba(abstracts.$primary-dark, 0.5);
+  justify-content: center;
+  align-items: center;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.modal-box {
+  background-color: abstracts.$primary-dark;
+  display: flex;
+  flex-direction: column;
+  color: white;
+  width: 700px;
+  height: 400px;
+}
+
+.modal-title {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  justify-content: flex-start;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.modal-footer {
+  display: flex;
+  align-self: flex-end;
+  justify-self: flex-end;
+  margin: 10px;
+}
+
+.modal-btn {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-bottom: 20px;
+}
+
+a {
```

Comment:
Tämä vaikuttaa kaikkiin linkkeihin koko projektissa. Yleiset linkkimuotoilut löytyy typographysta, jos haluaa tehdä niihin muutoksia. Jos tarkoitus on vaikuttaa vain modaalin linkkeihin, tämän voi nimetä joko ".modal-box > a" tai sen voi siirtää modal-boxin sisään näin:

```
.modal-box {
_tähän modaalin muotoilut_

   a {
   padding: 0;
   }
}
```